### PR TITLE
Fixing antifraud provider endpoints

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [X] Improvement (make an endpoint's title or description even better)
- [ ] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [X] No, but I am going to.

PR created to fix antifraud provider endpoints (EDU-8998).

### CODE REVIEW

File: VTEX - Antifraud Provider API.json [hunk 1]
```diff
@@ -2560,23 +2612,29 @@
...
                     "value": {
                         "type": "integer",
-                        "format": "int32"
+                        "format": "int32",
+                        "description": "VTEX transaction order value."
                     },
                     "ip": {
-                        "type": "string"
+                        "type": "string",
+                        "description": "The original ip address from browser."
```
Comment (karenkrieger) [inline VTEX - Antifraud Provider API.json:2629]:
```suggestion
                        "description": "The original IP address from the browser."
```

File: VTEX - Antifraud Provider API.json [hunk 2]
```diff
@@ -2560,23 +2612,29 @@
...
                     },
                     "store": {
-                        "type": "string"
+                        "type": "string",
+                        "description": "VTEX has a main Name for the store. This unique name is often used in url to access VTEX configurations."
```
Comment (karenkrieger) [inline VTEX - Antifraud Provider API.json:2633]:
```suggestion
                        "description": "VTEX has a main Name for the store. This unique name is often used in URL to access VTEX configurations."
```

File: VTEX - Antifraud Provider API.json [hunk 3]
```diff
@@ -2749,19 +2812,24 @@
...
                 "properties": {
                     "id": {
-                        "type": "string"
+                        "type": "string",
+                        "description": "VTEX transaction ID. The ID of this transaction must be in the response."
                     },
                     "tid": {
-                        "type": "string"
+                        "type": "string",
+                        "description": "Provider transaction ID. The ID of this transaction must be in the response. The antifraud provider MUST generates an unique transaction Id."
```
Comment (karenkrieger) [inline VTEX - Antifraud Provider API.json:2820]:
```suggestion
                        "description": "Provider transaction ID. The ID of this transaction must be in the response. The antifraud provider MUST generates a unique transaction Id."
```

**Review by karenkrieger — CHANGES_REQUESTED** [review]

File: VTEX - Antifraud Provider API.json (hunk 1 above)
Comment (ricardoaerobr) [reply VTEX - Antifraud Provider API.json:2629]: Fixed.

File: VTEX - Antifraud Provider API.json (hunk 2 above)
Comment (ricardoaerobr) [reply VTEX - Antifraud Provider API.json:2633]: Fixed.

File: VTEX - Antifraud Provider API.json (hunk 3 above)
Comment (ricardoaerobr) [reply VTEX - Antifraud Provider API.json:2820]: Fixed.

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**